.github/actions/setup-pip/action.yml

```diff
@@ -21,10 +24,12 @@ runs:
     - name: Install pip
       shell: bash
       run: |
-        pip install --no-cache-dir --upgrade "pip>=20.0.2"
+        make upgrade-pip
 
-    - name: Get pip cache dir
-      id: pip-cache
+    - name: Get pip cache dir & version
+      id: pip-info
       shell: bash
+      # pip versions before 20.1 do not have `pip cache`
       run: |
-        echo "::set-output name=pip-cache::$(pip cache dir)"
+        echo "::set-output name=pip-cache::$(pip cache dir 2>/dev/null || python -c "from pip._internal.locations import USER_CACHE_DIR; print (USER_CACHE_DIR)")"
```

Comment:
Are you OK if we just drop any pip version from before 20.1 and I can clean up this step?